app/src/main/java/com/iron/espresso/presentation/home/setting/SettingFragment.kt

```diff
@@ -20,6 +23,32 @@ class SettingFragment : Fragment() {
     ): View? {
         binding = DataBindingUtil.inflate(inflater, R.layout.fragment_setting, container, false)
         binding.lifecycleOwner = this
+
+        val settingCategoryList = arrayListOf<CategoryItem>()
+        for ((count, i) in resources.getStringArray(R.array.setting_category).withIndex()) {
+            val arrayList = arrayListOf<ItemList>()
+            when (count) {
+                0 -> for (j in resources.getStringArray(R.array.category_account)) {
+                    arrayList.add(ItemList(j))
+                }
+                1 -> for (j in resources.getStringArray(R.array.category_notice)) {
+                    arrayList.add(ItemList(j))
+                }
+                2 -> for (j in resources.getStringArray(R.array.category_info)) {
+                    arrayList.add(ItemList(j))
+                }
+                3 -> for (j in resources.getStringArray(R.array.category_etc)) {
+                    arrayList.add(ItemList(j))
+                }
+            }
+            settingCategoryList.add(CategoryItem(i, arrayList))
+        }
+
+        binding.settingRecyclerview.layoutManager = LinearLayoutManager(context)
+        binding.settingRecyclerview.adapter = context?.let { SettingAdapter(it,settingCategoryList) }
+
+        Glide.with(this).load(R.drawable.ic_launcher_background).circleCrop().into(binding.settingProfileImage)
```

Comment:
메소드 체이닝 컨벤션 - https://kotlinlang.org/docs/reference/coding-conventions.html#chained-call-wrapping
```suggestion
        Glide.with(this)
           .load(R.drawable.ic_launcher_background)
           .circleCrop()
           .into(binding.settingProfileImage)
```